cmd/minikube/cmd/dashboard.go

```diff
@@ -80,7 +82,13 @@ var dashboardCmd = &cobra.Command{
 			exit.WithError("Error getting client", err)
 		}
 
-		if _, err = api.Load(cc.Name); err != nil {
+		cp, err := config.PrimaryControlPlane(*cc)
+		if err != nil {
+			exit.WithError("Error getting primary control plane", err)
+		}
+
+		machineName := driver.MachineName(cc.Name, cp.Name)
```

Comment:
I feel like there is some room here for an API which is something like `cluster.Exists(profileName)`, `cluster.IsRunning(profileName)`, `cluster.IsHealthy(profileName)`. 

The dashboard command shouldn't need to know or care about individual machines. This abstraction is not going to scale if/when we allow multiple control plane VM's.

Reply:
This is correct, and I'm going to fix it in #6787